src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -25,25 +26,17 @@ namespace OpenLoco::Vehicles
     void vehiclePickupWater(OpenLoco::Interop::registers& regs);
     void sell(OpenLoco::Interop::registers& regs);
 
-    namespace Flags0C // commands?
+    enum class Flags38 : uint8_t
```

Comment:
What should we call Flags38?

Reply:
Think flags 38 will have to stay the name for now. Need to name more of them i think before coming up with a name